### ISSUE TITLE
Not able to lock recipes in the rolling machine

### ISSUE THREAD

**Issue by LukeSkywaIker** [TechReborn/TechReborn#1412]
Its not possible to lock recipes in the rolling machine in the latest build (2.12.1.446) . It always pulls every item out of the 3x3 grid. Therefore i´m not able to automate it i.e. with ae2.

Thanks!

**Comment by modmuss50** [comment]
The rolling machine really needs a rewrite, I did plan to make it work a bit like the auto crafting table.

**Comment by LukeSkywaIker** [comment]
Not before 1.13 ? ;)

Best regards!

**Comment by modmuss50** [comment]
it may happen before 1.13.

**Comment by modmuss50** [comment]
Ok, ive taken a look at this. First I tried to just add just a simple way of locking the recipe and that worked out fine. But the main issues arrises when trying to pump items in. 

The entire machine will need a rewrite to make this work as people want, but for now I have added a config to lock the recipe in place, by ensuring that there is at least 1 of each ingredient in the slots.

This config is off by default, and most likely wont help a lot of piping items in wont work great. I will leave this issue open however so I dont forget to rewrite it.
